hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/protocol/StorageContainerNodeProtocol.java

```diff
@@ -58,7 +60,8 @@
    */
   RegisteredCommand register(DatanodeDetails datanodeDetails,
                              NodeReportProto nodeReport,
-                             PipelineReportsProto pipelineReport);
+                             PipelineReportsProto pipelineReport,
+                             LayoutVersionProto layoutVersionInfo);
```

Comment:
Can you update javadoc here as we add new param layoutVersionInfo?

Reply:
yup